README.md

```diff
@@ -48,7 +48,7 @@ This directory is *not* automatically purged and may grow significantly when usi
 The test environment is configured in the file `~/.sonar/orchestrator/orchestrator.properties`:
 
     # Token used to download SonarSource private artifacts from https://repox.jfrog.io/repox
-    # Generate your API key at https://repox.jfrog.io/repox/webapp/#/profile
+    # Generate your API key at https://repox.jfrog.io/ui/admin/artifactory/user_profile
```

Comment:
If I follow that link, the UI is broken:
![image](https://user-images.githubusercontent.com/103252490/183422501-9e7eb788-d173-4df2-9ece-c429238a7f4f.png)

Maybe this should be reported to jfrog.

Reply:
good point, I've sent an email to Release Engineers